hvplot/util.py

```diff
@@ -750,3 +751,14 @@ def relabel_redim(hv_obj, relabel_kwargs, redim_kwargs):
     if redim_kwargs:
         hv_obj = hv_obj.redim(**redim_kwargs)
     return hv_obj
+
+
+def geoviews_is_available(raise_error: bool = False):
+    """
+    Check if GeoViews is available and raise an ImportError if not.
+    """
+    gv_available = find_spec('geoviews') is not None
+
+    if not gv_available and raise_error:
+        raise ImportError('GeoViews must be installed to enable the geographic options.')
```

Comment:
I got confused a minute here, does `find_spec('geoviews')` actually import GeoViews?

Reply:
No, but it checks if it is installed. 

![image](https://github.com/user-attachments/assets/5b1461b3-a2ce-466b-8e02-c25761b8dd8c)

Reply:
Yes ok that's what I thought. I was confused as then an `ImportError` is raised. Since importing GeoViews comes with side effects (registering lots of stuff), I'd like to make sure we're not removing an important step there. @hoxbro did you suggest `find_spec` for performance reasons?

Reply:
Just as an alternative and also easier to patch in the test suite.

I think it is okay to use `find_spec` when we check if it is available. We should import geoviews in the code, where there previously was a `try/except`, to get those juicy side effects.

Reply:
I also think the error message is wrong here. As it no longer only affects the UI code.